doc/man1/flux-shell.rst

```diff
@@ -0,0 +1,446 @@
+==============
+flux-shell(1)
+==============
+
+
+SYNOPSIS
+========
+
+**flux-shell** [*OPTIONS*] *JOBID*
+
+DESCRIPTION
+===========
+
+flux-shell(1), the Flux job shell, is the component of Flux which manages
+the startup and execution of user jobs. flux-shell(1) runs as the job user,
+reads the jobspec and assigned resource set R for the job from the KVS,
+and using this data determines what local job tasks to execute. While job
+tasks are running, the job shell acts as the interface between the parent
+instance and the job by handling I/O, signals, and finally collecting
+the exit status of tasks as they complete.
+
+The design of the Flux job shell allows customization through a set of
+builtin and runtime loadable shell plugins. These plugins are used to
+handle I/O redirection, PMI, CPU and GPU affinity, debugger support and
+more. Details of the flux-shell(1) plugin capabilities and design can be
+found in the PLUGINS section below.
+
+flux-shell(1) also supports configuration via a Lua-based configuration
+file, called the shell ``initrc``, from which shell plugins may be loaded
+or shell options and data examined or set. The flux-shell(1) initrc may
+even extend the shell itself via simple shell plugins developed directly
+in Lua. See the SHELL INITRC section below for details of the ``initrc``
+format and features.
+
+While the flux-shell(1) is a normal executable and may be run in
+standalone mode (``-s, --standalone``), this option and other command
+line options listed below are mainly used for testing purposes. Most
+Flux users will interact with flux-shell(1) through the execution of
+Flux jobs. Therefore, the OPTIONS section may be of less interest than
+the subsequent sections in this manual.
```

Comment:
Maybe this could be shortened to:
> Most Flux users will interact with flux-shell(1) indirectly through the execution of Flux jobs, however flux-shell(1) accepts the following options in standalone mode (``-s, --standalone``), for test purposes.

Reply:
That is much better, thanks!